lua/codeium/util.lua

```diff
@@ -39,7 +39,8 @@ function M.get_newline(bufnr)
 end
 
 function M.has_win32()
-  return vim.call("exists", "*win32") == 1
+	return vim.fn.has("win32")
```

Comment:
Unfortunately this always returns true. Per our discussion your vim.fn does not have this entry either so has function is not doing what you think it is doing.

Can you check the output of this call?
`:lua =vim.loop.os_uname().sysname`

Reply:
Oooooh fancy! `Linux` and `Darwin`. I will update the repo to os_uname tonight.